start-workspace-from-url.adoc

```diff
@@ -0,0 +1,85 @@
+:prod-short: Che
+
+= Starting a workspace from an URL
+
+After using getting started examples from the Dashboard users want to create workspaces from existing Git repositories.
+Create workspaces from a Git repository by entering a custom URL on the {prod-short} dashboard page.
```

Comment:
```suggestion
Create a workspace from a Git repository by entering a custom URL on the {prod-short} dashboard page.
```

Reply:
... assuming that only one workspace is created by entering a custom URL on the dashboard page.

Reply:
@max-cx by default there is a new workspace created on every click (with an option we can disable that)

Reply:
@benoitf thank you. I'll try doing this myself as the user then and will update this thread.